docs/sources/mimir/visualize/native-histograms/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+description: Learn how to visualize native histograms.
+keywords:
+  - query metrics
+  - native histogram
+  - prometheus
+  - grafana
+  - panels
+  - explore
+  - promql
+menuTitle: Native histograms
+title: Visualize native histograms
+weight: 100
+---
+
+# Visualize native histograms
+
+Prometheus native histograms is a data type in the Prometheus ecosystem that makes it possible to produce, store, and query a high-resolution [histogram](https://prometheus.io/docs/concepts/metric_types/#histogram) of observations.
+To learn more about the native histograms data type and how to start sending native histograms to Grafana Mimir,
+refer to [Send native histograms to Mimir]({{< relref "../../send/native-histograms" >}}).
+
+## Prometheus Query Language
+
+The Prometheus Query Language (PromQL) allows you to query native histogram metrics.
+PromQL queries of native histograms are different from those of classic histograms.
+
+For more information about PromQL, refer to [Querying Prometheus](https://prometheus.io/docs/prometheus/latest/querying/basics/).
+
+<!--
+Given an example, where a histogram metric is named `request_duration_seconds`,
+and it has some labels, you can apply an aggregation (`sum`) to the time series of the metric.
+Doing so produces a single output.
+ -->
+
+### Query your histogram’s count or sum
+
+To query the total count of observations within a histogram, use the following queries:
+
+```PromQL
+# Native histograms:
+histogram_count(sum(request_duration_seconds))
+
+# Classic histograms:
+sum(request_duration_seconds_count)
+```
+
+To query the total sum of observed values, use the following query:
+
+```PromQL
+# Native histograms:
+histogram_sum(sum(request_duration_seconds))
+
+# Previous classic histograms:
+sum(request_duration_seconds_sum)
+```
+
+### Find rate of observations
+
+To query the rate of all observations, calculated over 5 minute time window, use the following query:
+
+```PromQL
+# Native histograms:
+histogram_count(sum(rate(request_duration_seconds[5m])))
+
+# Previous classic histograms:
+sum(rate(request_duration_seconds_count[5m]))
+```
+
+In Grafana the range vector selector of `5m` would be replaced by `$__rate_interval`.
```

Comment:
```suggestion
```

Reply:
@krajorama, please make sure that this line is about the query example in https://github.com/grafana/mimir/pull/6231/files#diff-7b748ae7e809e51673da08b4c0a45231f198c3e1bbd892ca24e7c941ecce40d8R61-R67.